src/open_inwoner/templates/pages/ssd/yearly_reports_list.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'master.html' %}
+{% load i18n grid_tags %}
+
+{% block content %}
+    {% render_grid %}
+        {% render_column span=9 %}
+            <h1 class="h1" id="title">
+                {% trans "Mijn uitkeringen" %}
+            </h1>
+
+            <div class="tab--container">
+                <div class="tabs">
+                    <ul class="list tabs__headers">
+                        <li class="list-item tab__header--item"><a href="{% url 'profile:monthly_benefits_index' %}" id="yearly" class="link tab__header">{% trans "Maandspecificaties" %}</a></li>
+                        <li class="list-item tab__header--item"><a href="#jaaropgaven" id="yearly" class="link tab__header active">{% trans "Jaaropgaven" %}</a></li>
+                    </ul>
+
+                    <div class="tabs__body">
+                        <div id="jaaropgaven" class="tab__content active">
+                            <p>{% trans "Selecteer het jaar dat u wilt downloaden. Aan deze jaaropgaven kunt u geen rechten ontlenen." %}</p>
+                            <ul>
+                            {% for report in yearly_benefits %}
+                                <li>
+                                    <a href="{% url 'ssd:download_yearly_benefits' file_name=report %}" target="_blank" rel="noopener noreferrer">{{ report }}</a>
+                                </li>
```

Comment:
The namespace needs to be changed from `ssd` to `profile` since the urls are exposed via the CMS.